File: test/resources/HelloProphecy/prophecy/pipelines/gold_top_customers/code/gold_top_customers/graph/gold_total_sales_by_customer.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.utils import *
from prophecy.libs import typed_lit
from gold_top_customers.config.ConfigStore import *
from gold_top_customers.udfs.UDFs import *

def gold_total_sales_by_customer(spark: SparkSession) -> DataFrame:
    return spark.read.table("`scottdemo`.`gold_total_sales_by_customer`")

File: test/resources/HelloProphecy/prophecy_scala/pipelines/gold_top_customers/code/src/main/scala/io/prophecy/pipelines/gold_top_customers/graph/gold_total_sales_by_customer.scala
package io.prophecy.pipelines.gold_top_customers.graph

import io.prophecy.libs._
import io.prophecy.pipelines.gold_top_customers.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object gold_total_sales_by_customer {

  def apply(context: Context): DataFrame =
    context.spark.read.table("`scottdemoscala`.`gold_total_sales_by_customer`")

}
